watertap/data/techno_economic/zeroth_order_model_data/nano_filtration.yaml

```diff
@@ -0,0 +1,88 @@
+default:
```

Comment:
This is a duplicate of the one we already have in the `techno_economic` folder (which was used as the initial example. However, I am seeing some differences in the file, notably in the names of components (e.g. `Fe_3+` versus `iron`), the longform names and many values (likely due to precision).

We need to reconcile these two version and make sure we have standard names across all files (This also includes the water source definitions).

Reply:
Yes, I updated all the names based on standard defined names in the following document that was reviewed by everyone I believe https://app.box.com/s/kd19k78qgnfq2p8mvt3l617xa0tji0uw